16/umbraco-cms/fundamentals/setup/upgrading/README.md

```diff
@@ -1,11 +1,15 @@
 ---
-description: This is the guide for upgrading existing Umbraco installations.
+description: Select the guide that fits your upgrade situation and learn more about the recommended approach.
 ---
 
 # Upgrade your project
 
 In this section, you will find everything you need to upgrade your Umbraco CMS project.
```

Comment:
Can this be rephrased a bit.. a lot of you/your in a single sentence.

Maybe something like: In this section, you will find resources required to upgrade your Umbraco CMS project.